### DISCUSSION TITLE
Huawei P10 won't connect after bootloader flash.

### DISCUSSION THREAD

**Discussion by Haldrie** [kitsuned/PotatoNV#130] [Q&A]
I have been trying for a couple of days now to use this tool and every time it keeps timing out with my phone never reconnecting after the bootloader is flashed to it until I disconnect and reconnect the USB cable.
![potatonvfail](https://user-images.githubusercontent.com/16618994/176346161-d515ab7f-9f85-45ff-b3f5-faf7a77778b6.png)
.

**Comment by eh428077570** [comment]
you have not  enter the com1

**Reply by Haldrie** [reply]
Oh I forgot I posted this. Turns out all the tutorials I was watching (other then one I eventually found that was for something unrelated to bootloader unlocking) never mentioned reconnecting the battery after it got a connection which is what I ultimately had to do to get this to work. I guess USB power was getting reset along with the phone as it was trying to reboot in to fastboot mode and ultimately just lost power entirely as a result causing the issues I kept having.
